EndUserDocs/index.html

```diff
@@ -69,7 +69,15 @@ <h3><a id="getting-started">Getting Started</a></h3>
             <br />
             When you open a binary (whether individually or as a diff) you'll need to point to both the binary itself and its PDB. After you do, a tab gets opened where
             you can dig into more details. If you point to the PDB first, SizeBench will try to locate the binary - for the moment it only looks side-by-side with the
-            PDB, or it knows how to navigate the Windows build shares for Microsoft employees.
+            PDB, or it knows how to navigate the Windows build shares for Microsoft employees.<br />
```

Comment:
>or it knows how to navigate the Windows build shares for Microsoft employees [](http://example.com/codeflow?start=17&length=76)

I removed the windows build share path locator but forgot to update these docs, mind removing that while you're here?